keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1536,6 +1496,10 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - We have at least the start of a plan for the next steps (changing
   the default mode, deprecating the old backends, etc).
 
+- No UNRESOLVED sections in the KEP. (In particular, we have figured
```

Comment:
About Beta graduation criteria:

> The nftables mode has seen at least a bit of real-world usage.

Do we have that? Can you elaborate a bit on that in the KEP?